R/class-AbundanceData.R

```diff
@@ -40,10 +40,12 @@ check_abundance_data <- function(object) {
         msg <- paste("Samples in the sample metadata are not in the same order as in the abundance data.")
         errors <- c(errors, msg)
       }
+      if (setequal(names(sampleMetadata@data), c(record_id_col, ancestor_id_cols))) {
+        msg <- paste("The sample metadata only contains record ID and ancestor ID columns.")
```

Comment:
This should probably also include how to fix the error. "Add actual metadata variables" or something ... maybe you can figure out how to word it so that it doesn't sounds so sassy lol

Reply:
i think ill put 'hey idiot, theres no real data here.'